bulk-ingester/src/main/resources/application.yml

```diff
@@ -15,3 +15,4 @@ alfresco.bulk.ingest:
   repository:
     pageSize: 100
   endpoint: activemq:queue:bulk-ingester-events
+  namespace-prefixes-mapping: classpath:namespace-prefixes.json
```

Comment:
We should update the README to document this file and where customers need to put it.